doc/source/examples/00-systemcoupling/heating_tank_fmu_fmu.py

```diff
@@ -0,0 +1,226 @@
+
+# %%
+#
+# Download input files
+# ~~~~~~~~~~~~~~~~~~~~
+# Clear the downloads target directory (which is to be used as the
+# working directory). Download the FMU files, which define the
+# participant-specific setup information.
+
+examples.delete_downloads()
+fmu_file_heater = examples.download_file(
+    "thermostat.fmu", "pysystem-coupling/heating_tank_fmu/FMU"
+)
+fmu_file_tank = examples.download_file(
+    "heatingTank.fmu", "pysystem-coupling/heating_tank_fmu_fmu/FMU"
+)
+
+
+# %%
+# Launch System Coupling
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Launch a remote System Coupling instance and return a *client* object
+# (a ``Session`` object) that allows you to interact with System Coupling
+# via an API exposed into the current Python environment.
+syc = pysystemcoupling.launch()
+
+# %%
+# Create analysis
+# ---------------
+# Creating the analysis consists of accessing the ``setup`` API,
+# loading participants, creating and verifying both interfaces and
+# data transfers, querying for setup errors, and modifying settings.
+#
+# Access the ``setup`` API
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+setup = syc.setup
+
+
+# %%
+# Add participants
+# ~~~~~~~~~~~~~~~~~
+# Use ``add_participant`` to create ``coupling_participant`` objects
+# representing the FMU participants, based on the previously defined
+# setup information.
+heater_part_name = setup.add_participant(input_file=fmu_file_heater)
+tank_part_name = setup.add_participant(input_file=fmu_file_tank)
+
+
+# %%
+# FMU settings
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Change FMU parameters by accessing ``fmu_parameter``
+
+# Change the "maximum heat output" settings
+setup.coupling_participant[heater_part_name].fmu_parameter["Real_2"].real_value = 10.0
+
+# %%
+# Change the "target temperature" settings
+setup.coupling_participant[heater_part_name].fmu_parameter["Real_3"].real_value = 350.0
+
+# %%
+# Change the "heat scale factor" settings
+setup.coupling_participant[heater_part_name].fmu_parameter["Real_4"].real_value = 2.0
+
+# %%
+# Create interfaces and data transfers
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create interfaces and data transfers by specifying participant variables
+# to map to one another. This consists of calling the appropriate commands
+# to create an interface and both temperature and heat flow data transfers.
+
+# Create a coupling interface for tank <-> heater (sensor, heat source)
+interface = setup.add_interface(
+    side_one_participant=tank_part_name, side_two_participant=heater_part_name
+)
+
+# Create data transfer for "temperature"
+temperatureDataTransfer = setup.add_data_transfer(
+    interface=interface,
+    target_side="Two",
+    source_variable="Real_0",
+    target_variable="Real_0",
+)
+
+# Create data transfer for "heatflow"
+heatFlowDataTransfer = setup.add_data_transfer(
+    interface=interface,
+    target_side="One",
+    source_variable="Real_1",
+    target_variable="Real_1",
+)
+
+# %%
+# Change the ``time_step_size`` setting.
```

Comment:
are we changing the time step size or setting it? I think it's undefined until we set it?

Reply:
ditto the end time

Reply:
tbh, I'm not a huge fan of the comments from ln181 - ln199. None of the comments are terribly helpful. We're just stating directly what the command does without providing any context.

In contrast, the comment on ln164 *is* helpful as it's telling the user that `Real_0` is associated with the temperature.

Similarly, the comment on ln202 is helpful as it describes why we want to generate the csv chart info

Reply:
I think the 181-199 comments could just be bundled together as some general comment about configuring the settings for the analysis.